### PR TITLE
New Attribute binding doc

### PR DESCRIPTION
**Pull request by trekladyone**
Created a new Attribute binding doc.  Taken from the Attribute, class, and style bindings, which is being broken down into separate documents.

### CODE REVIEW

File: aio/content/guide/attribute-binding.md [hunk 1]
```diff
@@ -40,328 +32,41 @@ When the expression resolves to `null` or `undefined`, Angular removes the attri
 
 ## Binding ARIA attributes
 
-One of the primary use cases for attribute binding
-is to set ARIA attributes, as in this example:
+One of the primary use cases for attribute binding is to set ARIA attributes.
+
+To bind to an ARIA attribute, type the following:
```
Comment (jessicajaniuk) [inline aio/content/guide/attribute-binding.md:37]:
```suggestion
This is an example of how to bind to an ARIA attribute:
```

File: aio/content/guide/class-binding.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+The expression can be one of:
+
+* A space-delimited string of class names.
+* An object with class names as the keys and truthy or falsy expressions as the values.
+* An array of class names.
+
+With the object format, Angular adds a class only if its associated value is truthy.
+
+<div class="alert is-important">00
```
Comment (jessicajaniuk) [inline aio/content/guide/class-binding.md:31]: Did you mean to have this `00` here?

File: aio/content/guide/class-binding.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+Angular sets the property to the value of the bound expression, which is usually a string.  Optionally, you can add a unit extension like `em` or `%`, which requires a number type.
+
+<div class="alert is-helpful">
+
+To write a style in dash-case, type the following:
+
+<code-example language="html">&lt;nav [style.background-color]="expression"
+&gt;&lt;/nav&gt;
+
+To write a style in camelCase, type the following:
```
Comment (jessicajaniuk) [inline aio/content/guide/class-binding.md:106]: This line is in the middle of a code-example.

Additionally this is just my opinion, but `type the following` doesn't quite feel right for guides. Something like `Here is an example of how to write a style in camelCase` feels a bit more appropriate.

File: aio/content/guide/class-binding.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+
+  &lt;nav [style.backgroundColor]="expression"
+&gt;&lt;/nav&gt;</code-example>
```
Comment (jessicajaniuk) [inline aio/content/guide/class-binding.md:109]:
```suggestion
  &lt;nav [style.backgroundColor]="expression"&gt;&lt;/nav&gt;
</code-example>
```

File: aio/content/guide/class-binding.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+For example, set the ‘width’ style, type the following:  `[style.width]="width"`
+
+Angular sets the property to the value of the bound expression, which is usually a string.  Optionally, you can add a unit extension like `em` or `%`, which requires a number type.
+
+<div class="alert is-helpful">
+
+To write a style in dash-case, type the following:
+
+<code-example language="html">&lt;nav [style.background-color]="expression"
+&gt;&lt;/nav&gt;
```
Comment (jessicajaniuk) [inline aio/content/guide/class-binding.md:104]:
```suggestion
<code-example language="html">
    &lt;nav [style.background-color]="expression"&gt;&lt;/nav&gt;
```

File: aio/content/guide/class-binding.md (hunk 2 above)
Comment (mgechev) [inline aio/content/guide/class-binding.md:31]:
```suggestion
<div class="alert is-important">
```

**Review by mgechev — COMMENTED** [review]
Left a suggestion

**Pull request closed without merging**